src/Simulation/Simulators/ToffoliSimulator/Random.cs

```diff
@@ -1,36 +0,0 @@
-﻿// Copyright (c) Microsoft Corporation. All rights reserved.
```

Comment:
Why was Random removed, instead of updated like the rest? Did it get moved somewhere else?

Reply:
This was the C# implementation for a deprecated Q# operation. But the deprecated Q#, seen [here](https://github.com/microsoft/qsharp-runtime/blob/8a93f010dca6ab941ee25c1018cf7e1741fb88a0/src/Simulation/QSharpFoundation/Random.qs#L23), was updated to call into `DrawCategorical` which itself is intrinsic and has a separate implementation. So this can safely be removed instead of kept, since it is effectively dead code.